_data/accessibility-tests/footer.yml

```diff
@@ -0,0 +1,97 @@
+title: Footer
+component_status: pass
+test_items:
+# General tests
+  - summary: Some fields have auto-populate capability. 
+    summary_additional: When you navigate to an input field in the footer (like an email address), there will be an option in the browser to auto-populate information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
+  - summary: Footer font meets color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the font color and background color is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Footer elements are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, footer links and elements are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Footer icons and non-text content meet color contrast requirements. 
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the icon or other non-text elements and adjacent colors is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
```

Comment:
**note:** This was originally listed in the checklist as `1.4.3`. I updated in both locations to appropriately target `1.4.11`, [non-text color contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)